#goal definition
int32 traj_cmd
float64 x
float64 y
float64 z
float64 ori_w
float64 ori_x
float64 ori_y
float64 ori_z
string planGroup
---
#result definition
float64 x
float64 y
float64 z
float64 ori_w
float64 ori_x
float64 ori_y
float64 ori_z
int32 result_flag
---
#feedback
float64 x
float64 y
float64 z
float64 ori_w
float64 ori_x
float64 ori_y
float64 ori_z
